src/postman/transformers/server.ts

```diff
@@ -0,0 +1,10 @@
+import { IServer } from '@stoplight/types/dist';
+import { Url } from 'postman-collection';
+
+export function transformServer(url: Url): IServer | undefined {
+  if (!url.host) return;
+
+  return {
+    url: `${url.protocol}://${url.host.join('.')}${url.port ? `:${url.port}` : ''}`,
```

Comment:
Can we use `url.format`? It's going to be way safer

Reply:
Done, now `urijs` adds some protection.

Reply:
Well yeah, at least it moves the responsibility to somebody else :)